baselines/train/run_ray_train.py

```diff
@@ -96,6 +97,13 @@ def get_cli_args():
         help="Whether this script should be run as a test.",
   )
 
+  #parser.add_argument(
+  #      "--num_iterations",
+  #      type=int,
+  #      default=10,
+  #      help="The number of training iterations to run (lower=faster)"
+  #) #Todo: check this works, add to readme.
```

Comment:
Good idea, and should be straightforward to test too - did you want to do that as part of the current PR?

Reply:
I'm a fan of "branches should only do one thing".   So maybe merge the exp and do this later. 

actually, i did use it so reasonably confident it works. I just commented it out for the above reason.